#Base
logger/Event base
# GPS data
sensor_msgs/NavSatFix gps_data
# props
uint16 mision_id
